docusaurus/docs/cloud/advanced/upload.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Upload Provider
+displayed_sidebar: cloudSidebar
+description: Configure Strapi Cloud to use a third-party upload provider.
+canonicalUrl: https://docs.strapi.io/cloud/advanced/upload.html
+---
+
+# Upload Provider
+
+Strapi Cloud comes with a local upload provider out of the box. However, it can also be configured to utilize a third-party upload provider, if needed.
+
+:::prerequisites
+- A local Strapi project running on `v4.8.2+`.
+- Credentials for a third-party upload provider (see [Strapi Market](https://market.strapi.io/)).
+
+:::
+
+## Configuration
+To configure a third-party upload provider, in your Strapi project, create a `./config/env/production/plugins.js` or `./config/env/production/plugins.ts` file with the following content:
+
+```js
+module.exports = ({ env }) => ({
+  // ...
+  Required Upload Provider Plugin Configuration
+  // ...
+});
+```
+Each provider will have different configuration settings available. Review the respective entry for that provider in the [Marketplace](https://market.strapi.io/).
+
+Below are example configurations for the Upload plugins.
+
+<Tabs groupId="upload-examples" >
```

Comment:
Same suggestion as in the other doc: we could add a title with the name of the file.